src/Workspaces/CSharpTest/Formatting/FormattingTriviaTests.cs

```diff
@@ -1723,5 +1720,46 @@ public void NewLineOptions_LineFeedOnly()
 
             Assert.Equal(expected, actual);
         }
+
+        [WorkItem(4019, "https://github.com/dotnet/roslyn/issues/4019")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public void FormatWithTabs()
+        {
+
+            var code = @"#region Assembly mscorlib
+// C:\
+#endregion
+
+using System.Collections;
+
+class F
+{
+    string s;
+}";
+            var expected = @"#region Assembly mscorlib
+// C:\
+#endregion
+
+using System.Collections;
+
+class F
+{
+	string s;
+}";
+            var tree = SyntaxFactory.ParseCompilationUnit(code);
+
+            var text = DefaultWorkspace.Options.GetOption(FormattingOptions.NewLine, LanguageNames.CSharp);
+            var newLine = SyntaxFactory.ElasticEndOfLine(text);
+
+            tree = tree.ReplaceTokens(tree.DescendantTokens(descendIntoTrivia: true)
+                .Where(tr => tr.IsKind(SyntaxKind.EndOfDirectiveToken)), (o, r) => o.WithTrailingTrivia(o.LeadingTrivia.Add(newLine))
```

Comment:
Can you consider reformatting this statement?
- This line ends the first argument and begins the second argument, which makes it hard for me to visually parse. 
- It looks like the ```.Where``` might be a continuation of the ```tree.ReplaceTokens``` because of its indentation.